doc/classes/YSort.xml

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="YSort" inherits="Node2D" version="4.0">
 	<brief_description>
-		Sort all child nodes based on their Y positions.
+		[i]Deprecated.[/i] Sort all child nodes based on their Y positions. This has been replaced by the [code]y_sort_enabled[/code] property in [Node2D].
```

Comment:
```suggestion
		[i]Deprecated.[/i] Sort all child nodes based on their Y positions. This has been replaced by the [member Node2D.y_sort_enabled] property.
```

Reply:
Good point, I didn't realize that was the notation for that. Fixed